### PR TITLE
My Hello Microverse Project

### PR DESCRIPTION
**Pull request by anyars-encarta**
In this project, I added:

- [ ] Linters for HTML and CSS
- [ ] Index.html file
- [ ] index.css file
- [ ] .gitignore file
- [ ] test.md file
- [ ] Logo png file
- [ ] README file
- [ ] test.md to .gitignore file

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,183 @@
...
+```
+<!-- INSTALL -->
+
+### Install
+
+Install this project with the latest version of Git and a code editor.
+
+### Usage
+
+To run the project, execute the following command:
```
Comment (DuaneDave) [inline README.md:108]: To make your documentation professional, it is important to add in this section, how this app can be used, if there is none you add N/A.

File: .gitignore [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+
+# yarn v2
+.yarn/cache
+.yarn/unplugged
+.yarn/build-state.yml
+.yarn/install-state.gz
+.pnp.*
+
+# md file
+test.md
```
Comment (DuaneDave) [inline .gitignore]: Good job including test.md in this file, but to satisfy the requirements you'd need to include the node_modules directory as well in order to ensure it is always excluded in your commits.

**Review by DuaneDave — CHANGES_REQUESTED** [review]
Hi @anyars-encarta 👋, 

Good job so far on this project!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
✔️ Your PR summary is descriptive and clear enough

## Required Changes ♻️
- You have not set up the linters for this project as expected, consider visiting this [link](https://github.com/microverseinc/linters-config/tree/master/html-css) to set up the linters on this project in order to make it professional enough.

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- Consider making your commits history more professional by capitalizing the first letter of the sentence
 
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by skyv26 — CHANGES_REQUESTED** [review]
# Hi @anyars-encarta, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

## STATUS : REQUIRED CHANGES 🔴 



 ## Highlights
- Professional `README` ✅ 

 ## Required Changes ♻️
- [x] As your previous code reviewer mentioned about the linters setup and I can see you tried your best to add it but it is not working as expected it should. I am actually able to find out the issue that's why I attaching an image comparison below according to which your `linters.yml` should be inside the `workflows` folder.

### Problem Analyzer Table

<table>
<tr>
<th align=“middle”>Your linters config route</th>
<th align=“middle”>Required linter config route</th>
</tr>
<tr>
<td>

Screenshot from 2023-05-31 17-27-09


</td>
<td>

image

</td>
</tr>
</table>

_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @skyv26 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Shisui6 — APPROVED** [review]
Hi @anyars-encarta, 

# Approved 🪄
Your project is complete! There is nothing else to say other than... it's time to merge it 💎
Congratulations! 🎉 

## Highlights 🎯
- Descriptive PR title and summary ✅
- Follows Github flow ✅
- Good commit messages ✅

## Optional suggestions 🪶

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code. Some of them were simply missed by the previous reviewer, and addressing them will really improve your application._

You can also consider the following:

N/A

Cheers and Happy coding!🍻💻

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561), you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**